README.md

```diff
@@ -1 +1,70 @@
-# awesome-book
\ No newline at end of file
+# awesome-book
+
+> Awesome book ES6 Javascript.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+
+
+
```

Comment:
- [ ] Please add project description

Reply:
meaning I should add the functionality of the project